### PR TITLE
feat(content): add dinosaur dynasty achievement

### PR DESCRIPTION
**Pull request by CuriousMagpie**
Add dinosaur dynasty achievement.

### CODE REVIEW

File: website/common/locales/en/achievements.json [hunk 1]
```diff
@@ -147,5 +147,8 @@
   "achievementPolarProModalText": "You collected all the Polar Pets!",
   "achievementPlantParent": "Plant Parent",
   "achievementPlantParentText": "Has hatched all standard colors of Plant pets: Cactus and Treeling!",
-  "achievementPlantParentModalText": "You collected all the Plant Pets!"
+  "achievementPlantParentModalText": "You collected all the Plant Pets!",
+  "achievementDinosaurDynasty": "Dinosaur Dynasty",
+  "achievementDinosaurDynastyText": "Has hatched all standard colors of bird and dinosaur pets: Falcon, Owl, Parrot, Peacock, Penguin, Rooster, Pterodactyl, T-Rex, Triceratops, and Velociraptor!",
+  "achievementDinosaurDynastyModalText": "You colleted all the bird and dinosar pets!"
```
Comment (SabreCat) [inline website/common/locales/en/achievements.json]: Typo "dinosar"

Comment (SabreCat) [reply website/common/locales/en/achievements.json]: oh, also "colleted" XD

**Review by SabreCat — CHANGES_REQUESTED** [review]
One string has typos, and it looks like we're missing a needed migration.

File: website/common/locales/en/achievements.json (hunk 1 above)
Comment (CuriousMagpie) [reply website/common/locales/en/achievements.json]: Fixed. You know, English didn't used to have standard spelling. I'm just saying. XD

**Review by SabreCat — APPROVED** [review]

**Comment by SabreCat** [comment]
Merged in a7bde803499265bfcb056a37147c6c13d55bdb40 !

**Pull request closed without merging**